website/docs/r/mongodbatlas_secret_backend.html.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: "vault"
+page_title: "Vault: vault_mongodbatlas_secret_backend resource"
+sidebar_current: "docs-vault-resource-mongodbatlas-secret-backend"
+description: |-
+Creates an MongoDB Atlas secret backend for Vault.
+---
+
+# vault\_mongodbatlas\_secret\_backend
+
+Creates an MongoDB Atlas Secret Backend for Vault.
+
+~> **Important** All data provided in the resource configuration will be
+written in cleartext to state and plan files generated by Terraform, and
+will appear in the console output when Terraform runs. Protect these
+artifacts accordingly. See
+[the main provider documentation](../index.html)
+for more details.
+
+## Example Usage
+
+```hcl
+resource "vault_mongodbatlas_secret_backend" "config" {
+  backend           = "vault_mount.mongo.path"
+  private_key       = "privateKey"
+  public_key        = "publicKey"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `namespace` - (Optional) The namespace to provision the resource in.
+  The value should not contain leading or trailing forward slashes.
+  The `namespace` is always relative to the provider's configured [namespace](/docs/providers/vault#namespace).
+  *Available only for Vault Enterprise*.
```

Comment:
Not sure if this is needed, although I saw it listed in most of the docs?

Reply:
Yup! This is to be added to all docs. The `UpdateSchemaResource` wrapper you added in `provider.go` around the resource is what adds this schema field to all resources in the provider :)

Reply:
Ah, that makes sense. Thank you!